ButtonFunctions/update.py

```diff
@@ -17,13 +17,16 @@ def update(application):
         # check data
         pforce = float(pforce)
         force = pforce
-        time = float(time) / 1000
+        time = float(time)
         application.forces.append(force)
         application.times.append(time)
         ydata = np.asarray(application.forces)
         xdata = np.asarray(application.times)
         application.plot_pannels['plot2'].plot(xdata, ydata, clear=True, _callSync='off')
-
+        application.plot_pannels['plot2'].setTitle("Thrust Curve")
+        application.plot_pannels['plot2'].setLabel('left', 'Force (N)')
+        application.plot_pannels['plot2'].setLabel('bottom', 'Time (s)')
```

Comment:
I believe that this modification would be more appropriate in the Application class, cause is a configuration issue. The update class should concern about data issues.